aws-ts-eks/dashboard.ts

```diff
@@ -0,0 +1,315 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as k8s from "@pulumi/kubernetes";
+
+/**
+ * The Dashboard component creates a deployment of the Kubernetes Dashboard using the best practices listed at
+ * https://docs.aws.amazon.com/eks/latest/userguide/dashboard-tutorial.html.
+ */
+export class Dashboard extends pulumi.ComponentResource {
+    /**
+     * Creates an instance of the Dashboard component.
+     *
+     * @param name The _unique_ name of the component.
+     * @param opts A bag of options that control the behavior of this component and its children.
+     */
+    constructor(name: string, opts?: pulumi.ComponentResourceOptions) {
+        super("Dashboard", name, {}, opts);
```

Comment:
We normally use qualified names.

Reply:
Qualified to what, though? The stack name? This is a general issue with authoring components inside stacks, IMO.

Reply:
I think for any component, we'll have to pick appropriate package/module names, even if the component doesn't currently live in that package/module.

I thought that the engine (or some other parts of our system) enforced that these names be `X:Y:Z`.